pkg/specgen/generate/container.go

```diff
@@ -100,13 +99,9 @@ func CompleteSpec(ctx context.Context, r *libpod.Runtime, s *specgen.SpecGenerat
 	if err != nil {
 		return nil, err
 	}
-	// First transform the os env into a map. We need it for the labels later in
-	// any case.
-	osEnv, err := envLib.ParseSlice(os.Environ())
-	if err != nil {
-		return nil, errors.Wrap(err, "error parsing host environment variables")
-	}
 
+	rtc.Containers.EnvHost = s.EnvHost
```

Comment:
This doesn't seem right. We should not be modifying the runtime config once Podman is running - it's asking for all sorts of wierdness in `podman system service` when multiple calls come in at the same time.

Reply:
This is data specific to the container, not to the general config.

Reply:
I think I am mistaken, opened a PR for containers/common.